common/lib/xmodule/xmodule/js/fixtures/video_all.html

```diff
@@ -71,16 +71,23 @@
 
         <ul class="wrapper-downloads">
           <li class="video-tracks">
-            <div class="a11y-menu-container">
-                <a class="a11y-menu-button" href="#" title=".srt">.srt</a>
-                <ol class="a11y-menu-list">
-                  <li class="a11y-menu-item">
-                    <a class="a11y-menu-item-link" href="#txt" title="Text (.txt) file" data-value="txt">Text (.txt) file</a>
-                  </li>
-                  <li class="a11y-menu-item active">
-                    <a class="a11y-menu-item-link" href="#srt" title="SubRip (.srt) file" data-value="srt">SubRip (.srt) file</a>
-                  </li>
-                </ol>
+            <div class="wrapper-more-actions">
+              <button type="button" class="button button-more has-dropdown" aria-haspopup="true" aria-expanded="false" aria-controls="dropdown-transcript-format" aria-disabled="false">
+                <span class="icon fa fa-sort-desc" aria-hidden="true"></span>
+                <span class="sr">Transcript download format</span>
+              </button>
+              <ul class="dropdown-menu is-hidden" id="dropdown-transcript-format" tabindex="-1">
+                <li class="dropdown-item active">
+                  <a class="action" href="#srt" title="SubRip (.srt) file" data-value="srt" aria-disabled="false">
```

Comment:
I am generally against using the ```title``` attribute on anything other than an ```<iframe>```.  Contrary to popular belief, Title elements are not accessible to anyone other than mouse users.  Screen readers will read them, but only as a fallback, in cases where no other accessible text is available, and only if the user has their screen reader configured that way (not the default).  Since the title text here mirrors the accessible text, I would get rid of it.  Some accessibility checkers will flag it as a "needs manual verification" since they are want someone to verify why it is there.

Also, I also do not recommend specifying ```aria-disabled="false"``` here since that is the implied value:
https://w3c.github.io/aria-in-html/#aria-adds-nothing-to-default-semantics-of-most-html-elements